### PR TITLE
docs: 📝 update text by simplifying a bit

### LINKED ISSUE #14: Create draft "what will be covered" section in the syllabus

### PR DESCRIPTION
**Pull request by signekb**
## Description

This PR completes some minor text edits in the `syllabus.qmd`. 

Mainly, I have tried to remove words and sentences that are (/that I deem) unnecessary to try to keep things a bit simpler and more to the point :) 

I also fixed a layout issue with the nested lists. 

Closes #14 (this issue is the reason why I started going through this file, but it was almost filled out already)

## Reviewer Focus

This PR needs an in-depth review. 

## Checklist

- [X] Ran spell-check
- [X] Formatted Markdown
- [X] Rendered website locally

### CODE REVIEW

File: preamble/syllabus.qmd [hunk 1]
```diff
@@ -7,45 +7,55 @@
 Our overall learning outcome for the course is:
 
 1.  Describe some core features of effective team-based, collaborative
-    workflows and identify the components that make these workflows
-    effective compared to other workflows. Then use Git and GitHub with
-    approaches that strongly support effective collaboration.
+    workflows and use Git and GitHub with approaches that support
+    effective collaboration.
```
Comment (signekb) [inline preamble/syllabus.qmd:11]: It's not that I don't think the removed part is important, I'm just not sure I think it should be a part of the overall learning outcome. I think it was a bit "wordy"/long before :)

File: preamble/syllabus.qmd [hunk 2]
```diff
@@ -7,45 +7,55 @@
...
+    1.  reviewing a pull request,
+    2.  giving suggestions and feedback, and
+    3.  identifying how (and if) the changes should be merged into
+        project.
 
 ## Is this course for you? {#sec-is-it-for-you}
 
+This course will cover:
+
+-   Using Git and GitHub with VS Code and the GitHub web interface
```
Comment (signekb) [inline preamble/syllabus.qmd:48]: I added that we'll use the GitHub web interface - bc that's also what we do with all our work, and I think it'll make sense to use it in this course as well 👍

File: preamble/syllabus.qmd [hunk 3]
```diff
@@ -7,45 +7,55 @@
...
+-   Working with at least one other person on a project (*a Git
+    repository*)
```
Comment (signekb) [inline preamble/syllabus.qmd:50]: I added the "on a project (*a Git repository*) part here

File: preamble/syllabus.qmd [hunk 4]
```diff
@@ -7,45 +7,55 @@
...
+-   Working with plain text files (as Markdown `.md`)
+
+This course will **NOT** cover:
+
+-   Using any programming language (like R or Python)
+-   Any project management, for example related to tasks/issues (even
+    though we briefly cover issues) and kanban boards
```
Comment (lwjohnst86) [inline preamble/syllabus.qmd:57]:
```suggestion
    though we briefly cover issues) and Kanban boards
```

File: preamble/syllabus.qmd (hunk 1 above)
Comment (lwjohnst86) [reply preamble/syllabus.qmd:11]: Sounds fine with me :relaxed:

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]

**Review by lwjohnst86 — APPROVED** [review]

**Pull request merged**